arches_for_science/media/js/views/components/workflows/project-report-workflow/project-report-select.js

```diff
@@ -3,30 +3,93 @@ define([
     'arches',
     'knockout',
     'knockout-mapping',
+    'utils/resource',
     'templates/views/components/workflows/project-report-workflow/project-report-select.htm'
-], function(_, arches, ko, koMapping, projectReportSelectTemplate) {
+], function(_, arches, ko, koMapping, resourceUtils, projectReportSelectTemplate) {
     function viewModel(params) {
+        this.physicalThingPartOfSetNodeId = '63e49254-c444-11e9-afbe-a4d18cec433a';
+        this.physicalThingPhysicalPartofObjectNodeId = 'b240c366-8594-11ea-97eb-acde48001122';
+        this.physThingSetNodegroupId = 'cc5d6df3-d477-11e9-9f59-a4d18cec433a';
+
         var self = this;
+        let initializing = true;
         this.projectValue = ko.observable();
-        this.locked = ko.observable(false);
-        
-        this.saveValues = function(){ //save the savedData and finalize the step
-            params.form.savedData({
-                tileData: koMapping.toJSON(self.tile().data),
-                resourceInstanceId: self.tile().resourceinstance_id,
-                tileId: self.tile().tileid,
-                nodegroupId: self.tile().nodegroup_id,
-                fileTileData: ko.unwrap(self.fileTileData),
-            });
-            self.locked(true);
-            params.form.complete(true);
-            params.form.saving(false);
+        this.physicalThingsValue = ko.observable();
+        this.hasSetWithPhysicalThing = ko.observable();
+        this.setsThatBelongToTheProject = ko.observable();
+        this.projectNameValue = ko.observable();
+        this.locked = params.form.locked;
+
+        this.updateValues = function(val){
+            if (val !== null) {
+                self.projectValue(val.project);
+                self.physicalThingsValue(val.physicalThings);
+                self.setsThatBelongToTheProject(val.physicalThingSet);
+            }
         };
 
-        this.projectValue.subscribe(() => {
-            if (this.projectValue()) {
+        if (params.value()) {
+            this.updateValues(params.value());
+        }
+
+        this.projectValue.subscribe(function(val){
+            if (!initializing) {
+                self.physicalThingsValue(null);
+            }
+            initializing = false;
+            if (val) {
+                var res = resourceUtils.lookupResourceInstanceData(val);
+                res.then(
+                    function(data){
+                        self.projectNameValue(data._source.displayname);
+                        let setTileResourceInstanceIds;
+                        let setTile = data._source.tiles.find(function(tile){
+                            return tile.nodegroup_id === self.physThingSetNodegroupId;
+                        });
+                        if (setTile && Object.keys(setTile.data).includes(self.physThingSetNodegroupId) && setTile.data[self.physThingSetNodegroupId].length) {
```

Comment:
cosmetic suggestion if you like
```suggestion
                        if (setTile?.data?.[self.physThingSetNodegroupId]?.length) {
```